JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,29 @@
-//
-//  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
+
 // 과일 저장소 타입
 class FruitStore {
+    enum Fruit: CaseIterable {
+        case strawberry, banana, pineapple, kiwi, mango
+    }
+    
+    let defaultStock = 10
```

Comment:
`수량`이라는 점을 알 수 있는 좀 더 디테일한 변수명이면 좋을 것 같아요 ㅎㅎ

```suggestion
    let numberOfStock = 10
```

```suggestion
    let defaultNumberOfStock = 10
```

위와 같은 변수명들은 어떨까요?

Reply:
변경했습니다!